topics/coursera/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Andrew Ng, Daphne Koller
+display_name: Coursera
+github_url: https://github.com/topics/coursera
```

Comment:
This is more for an org page, or specific repo. I think it can be removed from here.
```suggestion
```